tracer/src/Datadog.Trace/ContinuousProfiler/ProfilerStatus.cs

```diff
@@ -27,6 +27,8 @@ public ProfilerStatus()
                 (fd.OSPlatform == OSPlatformName.Windows && (fd.ProcessArchitecture == ProcessArchitecture.X64 || fd.ProcessArchitecture == ProcessArchitecture.X86)) ||
                 (fd.OSPlatform == OSPlatformName.Linux && fd.ProcessArchitecture == ProcessArchitecture.X64);
 
+            _isProfilingEnabled = false;
```

Comment:
I'm wondering, what's the impact if the tracer _thinks_ the profiler is enabled and calls `EnsureNativeIsIntialized()` - does that "enabled" the profiler as expected? Is it as though the profiler _was_ always enabled in that case? 🤔

Reply:
Discussed offline - this is fine, though we may refactor at a later point to better represent the profiler status to the tracer (for purposes logging etc)